src/tiffimage_int.cpp

```diff
@@ -2043,7 +2044,7 @@ bool TiffHeaderBase::isImageTag(uint16_t /*tag*/, IfdId /*group*/, const Primary
 
 bool isTiffImageTag(uint16_t tag, IfdId group) {
   //! List of TIFF image tags
-  static constexpr TiffImgTagStruct tiffImageTags[] = {
+  static const std::unordered_set<TiffImgTagKey, TiffImgTagKey_hash> tiffImageTags({
```

Comment:
I was wondering why you chose `unordered_set` instead of `unordered_map` but I think there is something else even more relevant. I noticed that this is a constant hash table, and the `group` is always the same `ifd0Id` ... Probably we can remove that 2nd field from the table and just use the `uint16_t` saving some considerable space. 

Regarding the usage of `unordered_set` or `unordered_map`, I think that it makes sense to use `unordered_set` because you only have **keys** and not **values** in this table.

Reply:
Good point, I hadn't noticed that! Since this is all contained inside one function, I think a `switch` might be the best implementation.

Reply:
```
static const auto tiffImageTags = std::unordered_set<TiffImgTagKey, TiffImgTagKey_hash>{
```

is better I think. Avoids the )} change below as well.